var/spack/repos/builtin/packages/py-maestrowf/package.py

```diff
@@ -21,22 +21,28 @@ class PyMaestrowf(PythonPackage):
     version('master',  branch='master')
 
     # Pre-release candidates
-    version('1.1.5dev',    sha256='eb3d6f31c233e2cde3b84e15c657002b83ff43d4d6b218b33d023a4f527b9e08')
-    version('1.1.4dev1.0', sha256='67f59eed6fa69fc71b88a0a769de9f080300497d3c30d3a0893eabd0702bc48e')
-    version('1.1.4dev1.1', sha256='c8612b5423b44f11e2a7c4fbc31eb741013245870512ee2dbf7367024517528f')
+    version('1.1.7dev0', sha256='bcef838f13da396dd33cc7f503655de7a8f16ee5fe7b1e2a553044334a03f1f0')
```

Comment:
I assume we can remove this version now that 1.1.7 has an official release? Then we can remove `preferred=True` below.

Reply:
I plan to remove this in the future, but currently another package requires the dev version to function (at the very least it specifies it in their package file).